rose/client/car.py

```diff
@@ -1,5 +1,10 @@
+import logging
+import random
+import os
+from rose.common import config
 import component
 
+
```

Comment:
This is correct, but not related to import changes - other people are working on pep8 fixes, and such change will conflict with their changes, see #223.